README.md

```diff
@@ -1,16 +1,31 @@
-# Caution ⚠️
+# IceDOS
 
+This is a NixOS project that aims to create a highly opinionated gaming and general purpose computing experience, while still providing enough configuration options for my different needs.
+
+## Caution ⚠️
+
+- This repository is meant to be used on a fresh install of NixOS. Using it as a flake input is not currently in my needs, and thus not something that's tested or supported.
```

Comment:
The repository can be used on all NixOS setups, old or new. Instead, it is currently not compatible to be used as a flake input.

Reply:
Is it better now?